yesod-form/Yesod/Form/Fields.hs

```diff
@@ -267,9 +267,9 @@ $newline never
 passwordField :: Monad m => RenderMessage (HandlerSite m) FormMessage => Field m Text
 passwordField = Field
     { fieldParse = parseHelper $ Right
-    , fieldView = \theId name attrs val isReq -> toWidget [hamlet|
+    , fieldView = \theId name attrs _val isReq -> toWidget [hamlet|
```

Comment:
In such cases I prefer `_val` over `_` to keep the name. Anyway, it has to match the style of the rest of the project. Changed this to `_`.